src/mixs/schema/ancient.yml

```diff
@@ -52,6 +52,27 @@ enums:
     permissible_values:
       single:
       double:
+  SampAgeUnit:
+    permissible_values:
+      BCE:
+      CE:
+      BP:
+      cal BP:
+      cal BCE:
+      cal CE:
+      ka:
+      Ma:
+  SampAgeInferMethod:
```

Comment:
Why `Infer` and not `Inference`? 
I worry this 1337 c0der speek might trip up non-native speakers.

Reply:
UPDATE: I realise now that these are lists of allowed values. If end users don't really see this naming, I think this might be ok.

Reply:
Correct, just internal variable name :)